youtube_dl/extractor/shared.py

```diff
@@ -86,6 +94,14 @@ class VivoIE(SharedBaseIE):
         },
     }
 
+    def _extract_title(self, webpage):
+        data_title = self._search_regex(
+            r'data-name\s*=\s*(["\'])(?P<title>(?:.(?!\1))*.)\1', webpage,
+            'title', default=None, group='title')
+        if data_title:
+            return unescapeHTML(re.sub(r"\.[a-z0-9]{3,4}$", "", data_title))
+        return None
```

Comment:
Must not be None. Read coding conventions.

Reply:
Fixed in new commit.